examples/nodegroup-py/__main__.py

```diff
@@ -152,5 +152,20 @@
     opts=pulumi.ResourceOptions(providers={"kubernetes": cluster2_provider}),
 )
 
+eks.NodeGroup(
```

Comment:
Thank you! Do we need to assert anything here or the example will just fail if this doesn't provision?

Reply:
I'll see if there's an easy way to also add some assertion for sanity checking. Right now, it's just manual validation.